tests/run-make/rustdoc-verify-output-files/rmake.rs

```diff
@@ -0,0 +1,43 @@
+use std::fs::copy;
+use std::path::{Path, PathBuf};
+
+use run_make_support::{copy_dir_all, recursive_diff, rustdoc, tmp_dir};
+
+fn generate_docs(out_dir: &Path, json_output: bool) {
```

Comment:
Nit: *maybe* use a `JsonOutput::{Yes, No}` enum so the callsite can read like `generic_docs(&out_dir, JsonOutput::No);`? Feel free to ignore this, just a nit.

Reply:
It makes the code better so let's do it!